react/src/components/Products.js

```diff
@@ -7,23 +6,25 @@ import Loader from 'react-loader-spinner';
 import ProductCard from './ProductCard';
 import { useState, useEffect } from 'react';
 
-function Products({ frontendSlowdown, backend }) {
+function Products({ isJoin, frontendSlowdown, backend }) {
```

Comment:
I would prefer we not implement this in this PR since it's a feature change, not a refactor. I don't know whether the broader team will have any concerns about it so I don't want to sneak it into a PR that's meant to keep functionality the same and purely refactor from classes to function components.

(If we do end up doing it, I would also suggest we rewrite things to be `?slowdown=` and then have values `backend` (the backend slowdown), `frontend` (react only slowdown), `none` (join)) 

But my current opinion is to hold off on doing this and run it by others involved in demo like Neil/Kosty

Reply:
Understood. I've reverted the changes with https://github.com/sentry-demos/empower/pull/374/commits/6d9b95ac3d881c9a4419ed6d8fcb17b1296827cf.